### ISSUE TITLE
kivy.graphics documentation missing information

### ISSUE THREAD

**Issue by gkanarek** [kivy/kivy#8472]
On [the docs page for kivy.graphics](https://kivy.org/doc/stable/api-kivy.graphics.html), beginning with the BorderImage api documenation, there are many partial sentences, and some docstrings are entirely missing. I took a cursory glance at the .pyx files and they seem to be well documented, so there must be something going wrong when building the official docs.

**Comment by Julian-O** [comment]
Example, so we can tell when it is fixed:

`context_instructions.pyx` has the following declaration:

    cdef class ApplyContextMatrix(ContextInstruction):
        '''Pre-multiply the matrix at the top of the stack specified by
        `target_stack` by the matrix at the top of the 'source_stack'
    
        .. versionadded:: 1.6.0
        '''

Documentation says:

> class kivy.graphics.ApplyContextMatrix(**kwargs)[¶](https://kivy.org/doc/stable/api-kivy.graphics.html#kivy.graphics.ApplyContextMatrix)
>     Bases: [kivy.graphics.instructions.ContextInstruction](https://kivy.org/doc/stable/api-kivy.graphics.instructions.html#kivy.graphics.instructions.ContextInstruction)
>
> target_stack by the matrix at the top of the ‘source_stack’
>
> New in version 1.6.0.

The sentence "Pre-multiply the matrix at the top of the stack specified by" has gone missing.

[Is the problem related to the fact that it continues on the same line as the docstring opening?]

**Comment by AccelQuasarDragon** [comment]
had a friend try to compile kivy for arch linux and the compiled exe with Pyinstaller died, said something about  ModuleNotFoundError: No module named 'kivy.graphics.buffer'.

solution was to go to an older version of kivy

**Comment by FilipeMarch** [comment]
> had a friend try to compile kivy for arch linux and the compiled exe with Pyinstaller died, said something about  ModuleNotFoundError: No module named 'kivy.graphics.buffer'.
> 
> solution was to go to an older version of kivy
> edit: uninstalling an reinstalling kivy also worked 

Yeah true but I think it's not related to this issue, fortunately it's working now on my side
